src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -267,8 +296,9 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="_WasmSelectRuntimeComponentsForLinking" Condition="'$(WasmNativeWorkload)' == true" DependsOnTargets="_MonoSelectRuntimeComponents" />
-
+  <Target Name="_WasmSelectRuntimeComponentsForLinking" Condition="'$(WasmNativeWorkload)' == 'true'" DependsOnTargets="_MonoSelectRuntimeComponents" />
+  <Target Name="_WasmSelectRuntimeComponentsForLinking" Condition="'$(WasmNativeWorkload)' != 'true'" DependsOnTargets="_WasmSelectRuntimeComponentsForLinkingInTree" />
```

Comment:
We should include the mono targets `src/mono/nuget/Microsoft.NET.Runtime.MonoTargets.Sdk/Sdk` with the tests, like we do for other tasks, and targets. So, this would work automatically, and we wouldn't need to patch here.

Reply:
@radical could you please point out all the places ? I'm quite confused about it. There are probably few projects created from template during the unit test and I don't know if it belongs there or not. Or I don't know that they exist.

Reply:
https://github.com/dotnet/runtime/blob/main/src/mono/wasm/build/README.md#updating-dependencies-needed-for-building-wasm-apps

Reply:
We agreed with @radical to put back the temporary hack and keep https://github.com/dotnet/runtime/issues/61294 open for him to fix it properly.